src/AcmModal/AcmModal.tsx

```diff
@@ -1,45 +1,6 @@
 import React from 'react'
-import { Modal, Button, ModalVariant } from '@patternfly/react-core'
+import { Modal, ModalProps } from '@patternfly/react-core'
 
-/* istanbul ignore next */
-export interface IModalProps {
-    open: boolean
-    submit: () => void
-    cancel: () => void
-    title: string
-    message: string
-}
-
-/* istanbul ignore next */
-export const ClosedModalProps: IModalProps = {
-    open: false,
-    submit: () => {
-        /**/
-    },
-    cancel: () => {
-        /**/
-    },
-    title: '',
-    message: '',
-}
-
-export function AcmModal(props: IModalProps) {
-    return (
-        <Modal
-            variant={ModalVariant.medium}
-            title={props.title}
-            isOpen={props.open}
-            onClose={props.cancel}
-            actions={[
-                <Button key="confirm" variant="primary" onClick={props.submit}>
-                    Submit
-                </Button>,
-                <Button key="cancel" variant="link" onClick={props.cancel}>
-                    Cancel
-                </Button>,
-            ]}
-        >
-            {props.message}
-        </Modal>
-    )
+export function AcmModal(props: ModalProps) {
+    return <Modal {...props} ref={null} />
```

Comment:
What is `ref={null}` for?

Reply:
Seems like it's not covered in the type; so I'm not sure what to do with it for now besides give it something that will make it happy.  Do you think we will need to support a ref on the modals?